src/MongoDB.Driver/Encryption/ClientEncryption.cs

```diff
@@ -0,0 +1,159 @@
+﻿/* Copyright 2019-present MongoDB Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using MongoDB.Bson;
+using MongoDB.Libmongocrypt;
+
+namespace MongoDB.Driver.Encryption
+{
+    /// <summary>
+    /// Explicit client encryption.
+    /// </summary>
+    public class ClientEncryption : IDisposable
+    {
+        // private fields
+        private bool _disposed;
+        private readonly LibMongoCryptController _libMongoCryptController;
+
+        // constructors
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ClientEncryption"/> class.
+        /// </summary>
+        /// <param name="client">The client.</param>
```

Comment:
This should say what the client is used for (i.e. it's the client that contains the key vault collection.

However, maybe it should just be removed as it appears to duplicate a field in `ClientEncryptionOptions`